### ISSUE TITLE
Do not know how to remove items (or find items.yml)

### ISSUE THREAD

**Issue by MrMoodz** [Fabricio20/HubBasics#67]
Hello I am currently using hubbasics on a spigot 1.10.2 server and I love it, however, I would like to know how to remove the default items that come with joinitems. I created a custom bungee GUI, however, I cannot seem to be able to remove the default items from join items and I've been told to open the jar file and find the items.yml file. After doing that, all I found was important stuff I didn't want to change and a plugin.yml file. Please help? Just deleting the items from the main config doesn't seem to work. (And I want joinitems enabled for the bungee GUI)

**Comment by MrMoodz** [comment]
Nevermind, I have decided to just change the items instead of removing them, however, I now have a problem of not being able to use a compass for the server selector. Whenever I right-click the compass the message Command unknown pops up, and I think this is because I am using a custom GUI I made in another plugin called DeluxeMenus. I try using the command in-game and it works perfectly, however, linking it to an item doesn't seem to work. I also encountered this problem in another plugin, and I would like to say that I am not using this plugin's built-in GUI maker because I encountered some problems in that including: Players being able to enter stuff permanently into the GUI as if it were a chest and it displaying to any one else that opens the GUI, and they can take them out too.
